sdk/formrecognizer/ai-form-recognizer/samples/v5/javascript/README.md

```diff
@@ -0,0 +1,88 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-cognitive-services
+  - azure-form-recognizer
+urlFragment: ai-form-recognizer-javascript
```

Comment:
we can't change the library name, but we can change the url fragment?
```suggestion
urlFragment: ai-document-intelligence-javascript
```
what say?

Reply:
Since we're not renaming the package (yet at least) lets keep this the same for now.